src/main/scala/firrtl/transforms/TopWiring.scala

```diff
@@ -0,0 +1,247 @@
+// See LICENSE for license details.
+package firrtl.transform
+package TopWiring
+
+import firrtl._
+import firrtl.ir._
+import firrtl.passes._
+import firrtl.annotations._
+import firrtl.Mappers._
+import firrtl.graph._
```

Comment:
Probably best to avoid wildcard imports for new code, [per the Chisel3 style guide](https://github.com/freechipsproject/chisel3/blob/master/doc/style.md#imports).